en/rocky/8/guides/rocky-8-installation.md

```diff
@@ -68,159 +67,190 @@ Insert and boot off the installation medium (optical disk, USB flash drive, and
 
 Once booted, you'll be presented with a welcome splash screen.
 
-![Rocky Linux installation splash screen](images/installation-F01.png)
+![Welcome screen](images/installation-F01.png)
 
-If you do not press any key, the prompt will begin a countdown, after which the installation process will start by booting the highlighted Test This Media & Install Rocky Linux 8 ... option. You can also press enter to start the process immediately.
+If you do not press any key, the prompt will begin a countdown, after which the installation process will start by booting the highlighted `Test this media & install Rocky Linux 8` ... option. You can also press <kbd>enter</kbd> to start the process immediately.
 
 A quick media verification step will take place. This media verification step can save you the trouble of starting the installation only to find out halfway through that the installer will abort because of bad installation media.
 
 After the media check runs to completion, and the media is successfully verified to be usable, the installer will automatically continue to the next screen.
 
-Select the language you want to use to perform the installation in this screen. For this example, we select English (United States). Then click the Continue button.
+Select the language you want to use to perform the installation in this screen. For this example, we select _English (United States)_. Then click the <kbd>Continue</kbd> button.
 
-**Warning:** :warning: You'll next see warning screen. The screen will warn you that you are installing pre-release software and that you should NOT use the operating system in a production environment. If you okay with this, click `I want to proceed` to continue 
+> **WARNING:**
+:warning: You'll next see warning screen. The screen will warn you that you are installing pre-release software and that you **should NOT** use the operating system in a production environment. If you okay with this, click <kbd>I want to proceed</kbd> to continue.
 
 # Installation Summary
 
-The Installation Summary screen is an all-in-one area where you make the important decisions about the operating system to be installed. The screen is roughly divided into the following sections:
-    • Localization (Keyboard, Language Support, and Time & Date)
-    • Software (Installation Source and Software Selection)
-    • System (Installation Destination and Network & Hostname)
+The _Installation Summary_ screen is an all-in-one area where you make the important decisions about the operating system to be installed.
 
-We delve into each of these sections next and make changes where necessary.
+The screen is roughly divided into the following sections:
 
+- _Localization_: (Keyboard, Language Support, and Time & Date)
+- _Software_: (Installation Source and Software Selection)
+- _System_: (Installation Destination and Network & Hostname)
+
+We delve into each of these sections next and make changes where necessary.
 
 ## Localization Section
```

Comment:
Overall, I think either method works fine. Again, this wasn't my original document, so I'd have to defer to @wsoyinka

Reply:
I think we can use more lists to highlight like this time the choice to do.
Again, parts of the UI are in italic.